src/framework/theme/styles/core/_breaking-notice.scss

```diff
@@ -0,0 +1,25 @@
+$nb-breaking-notice-enable: true !global;
+
+@function nb-breaking-notice-show($theme) {
+
+  @debug '$nb-breaking-notice-enable';
+  @debug $nb-breaking-notice-enable;
+  @debug '$nb-breaking-notice-enable';
+
+  @if($nb-breaking-notice-enable) {
+    /* stylelint-disable string-no-newline */
+    @warn '
+           🔥 🔥 🔥 🔥 🔥 🔥  Nebular Attention 🔥 🔥 🔥 🔥 🔥 🔥
+
+           Starting with version 4.0, Nebular will be upgrading its Theme System.
+           If you use CUSTOM Nebular THEMES
+           please LET US KNOW AT http://bit.ly/2THkjSE
+
+           To supress this warning set $nebular-breaking-notice-enable: false; right before nb-install-global() call.
+
+
+           🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥';
```

Comment:
Could you please remove two extra 🔥 ?

Reply:
I hope you are kidding :)
Those extra are necessary to alight these rows vertically.